packages/domain-picker/src/components/index.tsx

```diff
@@ -160,11 +161,17 @@ const DomainPicker: FunctionComponent< Props > = ( {
 		retryRequest: retryDomainSuggestionRequest,
 	} = useDomainSuggestions( domainSearch.trim(), quantityExpanded, domainCategory, locale ) || {};
 
-	// filter out the free sub-domain from suggestions (1st position in the suggestions Array) when existingSubdomain prop has value
-	const domainSuggestions = allDomainSuggestions?.slice(
-		existingSubdomain ? 1 : 0,
-		isExpanded ? quantityExpanded : quantity
-	);
+	// filter out the free sub-domain suggestion when existingSubdomain prop has value
+	const domainSuggestions = existingSubdomain
+		? allDomainSuggestions
+				?.filter( ( suggestion ) => ! suggestion.is_free )
+				.slice(
+					0,
+					isExpanded
+						? quantityExpanded - FREE_DOMAIN_SUGGESTIONS_COUNT
+						: quantity - FREE_DOMAIN_SUGGESTIONS_COUNT
```

Comment:
Since we always use the constants to define `quantitiyExpanded` && `quantity` (we never overrule them by passing props to `<DomainPicker/>`), why don't we lower these values? In that way, we don't have to introduce another constant.

Reply:
> we never overrule them by passing props to <DomainPicker/>

Actually, we do. Check out the usage in `@automattic/launch` when displaying DomainPicker in Summary. However, I think looks more clean and readable with https://github.com/Automattic/wp-calypso/pull/51806/commits/041be4940834cdaebb7beed2e437e91d96735b33